### ISSUE TITLE
As a user I want an easy way to select my current project

### ISSUE THREAD

**Issue by avlisodraude** [beeldengeluid/labo-components#388]
As a user I want an easy way to selected my current project.

Acceptance criteria:
- [x] Replace current project btn with a dropdown list of the current available project.
- [x] Allow the user to search through the list of project via a search field/box.
- [x] Allow the user to create a new project (+ New Project).
- [x] Once a new project has been created this one should be the current selected project.

**Comment by avlisodraude** [comment]
**@lilimelgar @jblom** 

Issue for labo-components done in branch:
_issue/replace-projectsModal-by-list-in-singleSearch_

### What's been done?
All criteria point have been done

### Screenshot
![Screenshot 2019-05-07 at 16.37.30.png](https://images.zenhubusercontent.com/5a1e745d8a75884b90892ac4/74d69757-0811-4e8a-8910-81d4fdc659ce)

### Screencast Demo:
https://drive.google.com/open?id=1gTTor-dRUy8eMaDabXplX8Omtd6rZTEx

**Comment by avlisodraude** [comment]
**@jblom**
 
### Pull request created at:
https://github.com/beeldengeluid/labo-components/pull/390

**Comment by jwassenaarbg** [comment]
I really like this feature. The click- and then in the modal window- selection box was unnecessary, this is how it should be :)

Two things functionality wise: 
- The dropdown resizes while typing in a filter, I think it would be better if it stays the same size
- The file icon suggest you can save something, while it doesn't do anything - could this be removed? Or is it then unclear for users that one can select a project there?

In the code, on line 31 in Toolheader.jsx, you commented out a line. Can you remove this line completely? Or is there a reason to save the line for later? Other than that it looks good to me

**Comment by lilimelgar** [comment]
Based on the screencast (I cannot test branches yet) it all looks good from the user side.

**Comment by jblom** [comment]
@avlisodraude @jwassenaarbg @lilimelgar it works on the single search, but not on the resource viewer and compare tool ;-)

Besides this there were a few code details I noticed and fixed. (one big thing here is that I completely got rid of the project selector pop-up, which you still get when you want to bookmark, but haven't selected a project yet. Now I hide the search results checkboxes when no project is selected as well)

Current status: the single search is ok now; the resource viewer also (I think), but the compare project selector I haven't touched yet. So there is still some work to do there, which I'll finish later on.

**Comment by jblom** [comment]
@avlisodraude @lilimelgar ok I've finally merged this one. It should be ok now, but it requires proper testing, since I changed so many things.

Here you see that the checkboxes are gone when no program is selected. @lilimelgar should/could think about a nice solution to convey to the user that a project needs to be selected first.

![Schermafbeelding 2019-06-25 om 11 31 07](https://user-images.githubusercontent.com/917310/60087323-11dce800-973d-11e9-9235-6caf636a0779.png)
